### PR TITLE
Added layer 60, android TL scheme and merge script

### PR DESCRIPTION
**Pull request by danog**

### CODE REVIEW

**Comment by stek29** [comment]
Thanks, but why so many empty lines :(

**Comment by danog** [comment]
readability++++++

**Comment by stek29** [comment]
@danog I'd say readability---- tbh
But since I don't plan to maintain this repo in current state (not in my supercool json self-invented shit) I'm ok with it

But wtf is android.tl?

**Comment by stek29** [comment]
also pls squash your typo fix

File: end-to-end.tl [hunk 1]
```diff
@@ -1,3 +1,4 @@
+--types---
```
Comment (stek29) [inline end-to-end.tl]: aaaaaaaA_A_A_A_A_A_A_A_A__ASF_SDF_SD_FSD_FSD

**Comment by stek29** [comment]
k thx bye added to misc

**Pull request closed without merging**